### ISSUE TITLE
Extraneous Reblog links

### ISSUE THREAD

**Issue by marked** [ArchiveTeam/tumblr-grab#9]
Depends on style sheet but seen this on more than one.
Copy-Paste from a .warc.gz's WARC-Target-URI field.

http://9volt-art.tumblr.com/reblog/180424067376/Y6WsBSvg
http://9volt-art.tumblr.com/reblog/180424067376/Y6WsBSvg
https://www.tumblr.com/register?redirect_to=%2Freblog%2F180424067376%2FY6WsBSvg
https://www.tumblr.com/register?redirect_to=%2Freblog%2F180424067376%2FY6WsBSvg
http://9volt-art.tumblr.com/reblog/180359806016/auRYTNkl
http://9volt-art.tumblr.com/reblog/180359806016/auRYTNkl
https://www.tumblr.com/register?redirect_to=%2Freblog%2F180359806016%2FauRYTNkl
https://www.tumblr.com/register?redirect_to=%2Freblog%2F180359806016%2FauRYTNkl

**Comment by kiska3** [comment]
These links should be ignored in its entirety since we don't care about rebloging it

**Comment by marked** [comment]
On this crawl-
10% of hits are these reblog link and
6% of storage

```
Report on 9volt-art.tumblr.com
with 437 posts

Bytes Hostname  -------------------------------------------
       66.media.tumblr.com :  49 % =  310527495 /  629078388
       ve.media.tumblr.com :   1 % =    3792269 /  629078388
         static.tumblr.com :   0 % =     261006 /  629078388
                   gnu.org :   0 % =       1123 /  629078388
  sb.scorecardresearch.com :   0 % =        876 /  629078388
      consent.cmp.oath.com :   0 % =      26262 /  629078388
      9volt-art.tumblr.com :  37 % =  232563216 /  629078388
         assets.tumblr.com :   7 % =   45076272 /  629078388
        cdn.ampproject.org :   0 % =     425157 /  629078388
            www.tumblr.com :   6 % =   36404712 /  629078388

Bytes RootDir  -------------------------------------------
                    tagged :   4 % =   27035764 /  629078388
                     fonts :   2 % =   10972674 /  629078388
                       pop :   4 % =   26088561 /  629078388
                   archive :   3 % =   18582851 /  629078388
                      page :   1 % =    8216167 /  629078388
         register?redirect :   6 % =   35029575 /  629078388
                     notes :   2 % =   12566294 /  629078388
                      post :  24 % =  149163850 /  629078388

Hits Hostname  -------------------------------------------
       66.media.tumblr.com :  24 % =       4187 /      17131
       ve.media.tumblr.com :   0 % =          2 /      17131
         static.tumblr.com :   0 % =         24 /      17131
  sb.scorecardresearch.com :   0 % =          4 /      17131
                   gnu.org :   0 % =          2 /      17131
      consent.cmp.oath.com :   0 % =          4 /      17131
      9volt-art.tumblr.com :  56 % =       9660 /      17131
         assets.tumblr.com :   9 % =       1506 /      17131
            www.tumblr.com :  10 % =       1736 /      17131
        cdn.ampproject.org :   0 % =          6 /      17131


Hits RootDir  -------------------------------------------
                    tagged :   5 % =        780 /      17131
                     fonts :   2 % =        366 /      17131
                       pop :   4 % =        746 /      17131
                    oembed :   5 % =        864 /      17131
                    reblog :   5 % =        864 /      17131
                    mobile :   1 % =        182 /      17131
                    images :   2 % =        322 /      17131
                   archive :   4 % =        644 /      17131
         register?redirect :   5 % =        864 /      17131
                     notes :   6 % =        976 /      17131
                      page :   1 % =        178 /      17131
                     image :   3 % =        438 /      17131
                      post :  30 % =       5204 /      17131
```

**Comment by marked** [comment]
Tested PR 18
Fix confirmed
by checking .warc.gz.

Attaching report
```
Report on 9volt-art.tumblr.com
with 437 posts

Bytes Hostname  -------------------------------------------
      9volt-art.tumblr.com :  44 % =  213553630 /  484714724
  sb.scorecardresearch.com :   0 % =        610 /  484714724
       ve.media.tumblr.com :   1 % =    3792269 /  484714724
            www.tumblr.com :   0 % =    1264686 /  484714724
      consent.cmp.oath.com :   0 % =      26262 /  484714724
       66.media.tumblr.com :  55 % =  265816261 /  484714724
         static.tumblr.com :   0 % =     261006 /  484714724

Bytes RootDir  -------------------------------------------
                     notes :   2 % =   11945101 /  484714724
                      post :  30 % =  145457738 /  484714724
                   archive :   4 % =   17891941 /  484714724
                    tagged :   5 % =   25287432 /  484714724

Hits Hostname  -------------------------------------------
         static.tumblr.com :   0 % =         24 /      12200
      consent.cmp.oath.com :   0 % =          4 /      12200
       66.media.tumblr.com :  27 % =       3328 /      12200
            www.tumblr.com :   7 % =        848 /      12200
  sb.scorecardresearch.com :   0 % =          4 /      12200
       ve.media.tumblr.com :   0 % =          2 /      12200
      9volt-art.tumblr.com :  65 % =       7990 /      12200


Hits RootDir  -------------------------------------------
                     notes :   8 % =        926 /      12200
                     image :   4 % =        438 /      12200
                      post :  40 % =       4928 /      12200
                   archive :   5 % =        612 /      12200
                    tagged :   6 % =        728 /      12200
                    oembed :   7 % =        844 /      12200

posts:437	URIs:12200	bytes:484714724	URI_ratio:27	bytes_ratio:1109187
```

**Comment by marked** [comment]
Fix is PR #18
